cli/ballerina-cli/src/main/resources/new_cmd_defaults/README.md

```diff
@@ -1,5 +1,68 @@
-Prints "Hello, World!" with a hello function.
-[//]: # (above is the package summary)
-
 # Package Overview
-Prints "Hello, World!" as the output to the command line using a hello function.
+
+The `user/socialMedia` package provides a simple utility function to print "Hello, World!" to the command line. Additionally, it supports personalized greetings by accepting an optional name parameter.
+
+## Functionality
+
+The `hello` function is the primary utility provided by this package. It accepts an optional `string` parameter and returns a greeting message as a `string`.
+
+```ballerina
+public function hello(string? name) returns string {
+    if name !is () {
+        return string `Hello, ${name}`;
+    }
+    return "Hello, World!";
+}
+```
+
+### Parameters
```

Comment:
We dont need the parameter and return information. The API doc will be generated when publishing to the Ballerina Central.